app/components/dashboard/dashboard.component.js

```diff
@@ -11,195 +10,192 @@ import { LoadScript } from "@react-google-maps/api";
+    const { isLoading, error } = this.props;
 
     let total = teams
       .map((team) => team.totalDistance)
       .reduce((a, b) => a + b, 0);
 
     return (
-      <div className="dashboard">
-        <Segment loading={isLoading} className="secondary">
-          <div className="counter-uk">
-            <div className="wrapper">
-              <span className="text">Overal Distance: </span>
-              <span className="total">{total}km</span>
+      !error && (
+        <div className="dashboard">
+          <Segment loading={isLoading} className="secondary">
+            <div className="counter-uk">
+              <div className="wrapper">
+                <span className="text">Overal Distance: </span>
+                <span className="total">{total}km</span>
+              </div>
             </div>
-          </div>
-          <LoadScript
-            googleMapsApiKey={process.env.REACT_APP_GOOGLE_MAPS_API_KEY}
-            libraries={["geometry"]}
-          >
-            <MapUK teams={teams} />
-          </LoadScript>
-        </Segment>
-        <Segment loading={isLoading} className="primary">
-          <Grid container stackable columns={2} verticalAlign="middle">
-            <Grid.Row>
-              <Grid.Column>
-                <Header size="medium">Team Leaderboard</Header>
-                <div className="search-container">
-                  <Search
-                    input={{ icon: "search", iconPosition: "left" }}
-                    fluid
-                    loading={isLoading}
-                    onResultSelect={this.handleResultSelect}
-                    onSearchChange={debounce(this.handleSearchChange, 500, {
-                      leading: true,
-                    })}
-                    results={leaderboard}
-                    value={searchString}
-                    placeholder={"Search Team Name"}
+            <LoadScript
+              googleMapsApiKey={process.env.REACT_APP_GOOGLE_MAPS_API_KEY}
+              libraries={libraries}
+            >
+              <MapUK teams={teams} />
+            </LoadScript>
+          </Segment>
+          <Segment loading={isLoading || isSearchLoading} className="primary">
+            <Grid container stackable columns={2} verticalAlign="middle">
+              <Grid.Row>
+                <Grid.Column>
+                  <Header size="large">Team Leaderboard</Header>
+                  <div className="search-container">
+                    <Search
+                      input={{ icon: "search", iconPosition: "left" }}
+                      fluid
+                      loading={isSearchLoading}
+                      onSearchChange={this.handleSearchChange}
+                      value={value}
+                      placeholder={"Search Team Name"}
+                    />
+                  </div>
+                  <TeamLeaderboardTable
+                    isLoading={isSearchLoading}
+                    height={580}
```

Comment:
I know it is unlikely to change but should the specific height, width, padding values be moved into style.scss?

Reply:
For this component yes absolutely, I have moved it to style.css, for the sports leaderboards we need different heights so I'd leave that customisable.